src/compas_timber/ghpython/components/CT_Assembly/code.py

```diff
@@ -125,15 +125,22 @@ def RunScript(self, Beams, JointRules, Features, MaxDistance, CreateGeometry):
             for f_def in features:
                 beams_to_modify = self._get_copied_beams(f_def.beams)
                 for beam in beams_to_modify:
-                    beam.add_feature(f_def.feature)
+                    beam.add_features(f_def.feature)
 
         Geometry = None
         scene = Scene()
+        debug_scene = Scene()  # to collect debug related information
+
         if CreateGeometry:
```

Comment:
taking with fabio and jonas, we said we wanted to show the beams without features from the assembly component when there are no joint inputs or when `CreateGeometry` is False. I implemented this in [my latest PR](https://github.com/gramaziokohler/compas_timber/pull/177). I dont think it should cause any probelms.